packages/databricks-vscode/src/cli/CliWrapper.test.ts

```diff
@@ -204,4 +209,50 @@ nothing = true
             assert.equal(profiles.length, 0);
         });
     });
+
+    it("should set required env vars to the bundle run CLI calls", async () => {
+        const logFilePath = getTempLogFilePath();
+        const cli = createCliWrapper(logFilePath);
+        const authProvider = new ProfileAuthProvider(
+            new URL("https://test.com"),
+            "PROFILE",
+            true
+        );
+        const workspaceFolder = Uri.file("/test/123");
+        const runCmd = cli.getBundleRunCommand(
+            "dev",
+            authProvider,
+            "resource-key",
+            workspaceFolder
+        );
+        const expected = {
+            args: ["bundle", "run", "--target", "dev", "resource-key"],
+            cmd: cli.cliPath,
+            options: {
+                cwd: "/test/123",
+                env: {
+                    /* eslint-disable @typescript-eslint/naming-convention */
+                    DATABRICKS_CLI_UPSTREAM: "databricks-vscode",
+                    DATABRICKS_CLI_UPSTREAM_VERSION: extensionVersion,
+                    DATABRICKS_CONFIG_PROFILE: "PROFILE",
+                    DATABRICKS_HOST: "https://test.com/",
+                    DATABRICKS_LOG_FILE: logFilePath,
+                    DATABRICKS_LOG_FORMAT: "json",
+                    DATABRICKS_LOG_LEVEL: "debug",
+                    DATABRICKS_OUTPUT_FORMAT: "json",
+                    HOME: process.env.HOME,
+                    PATH: process.env.PATH,
+                    /* eslint-enable @typescript-eslint/naming-convention */
+                },
+                shell: true,
+            },
+        };
+        try {
+            assert.ok(isMatch(runCmd, expected));
+        } catch (e) {
+            // Run this in the "catch" case to show better error messages
+            assert.deepStrictEqual(runCmd, expected);
+            throw e;
+        }
```

Comment:
why use `isMatch` at all?

Reply:
I just don't want the tests to fail if we add more env vars, doing soft isMatch will prevent that. But yeah, the benefits are questionable to be honest